frontend/app/model/Eventbrite.scala

```diff
@@ -187,6 +187,7 @@ object Eventbrite {
       case t: InternalTicketing => t
     }
 
+    val ticketsNotSold = capacity - ticket_classes.map(_.quantity_sold).sum
```

Comment:
`val isLimitedAvailability = capacity - ticket_classes.map(_.quantity_sold).sum <= 15` instead maybe? Removes the logic from the template?